app.js

```diff
@@ -1,23 +1,42 @@
 const express = require('express');
 const { getTopics } = require('./controllers/controllers.topics');
-const { patchArticle } = require('./controllers/controllers.articles');
+const {
+  patchArticle,
+  getArticle,
+} = require('./controllers/controllers.articles');
+const res = require('express/lib/response');
```

Comment:
Don't need this line!

Reply:
VScode keeps putting that in, I've no idea why. All cleaned up!